styles/styles.css

```diff
@@ -909,6 +920,15 @@ main .section.bg-primary h6 {
   margin-bottom: 0;
 }
 
+/* section - no-padding-top for newsroom h1 */
```

Comment:
Rather than making this a newsroom specific style, you can introduce a global variant for headers like
```
 .header-no-top-padding.h1 {
    padding-top: 0;
}
```
This can then be useful for others as well.

Reply:
Not able to leverage global mixins since it was overcome by other set of css styles - 

<img width="1702" alt="image" src="https://github.com/hlxsites/sunstar/assets/37831850/fcb08eed-167d-4aac-be92-d7b0e7066512">

Reply:
Not able to leverage Global mixins since other CSS styles are nullifying it - 

<img width="1369" alt="image" src="https://github.com/hlxsites/sunstar/assets/37831850/48f8a457-ac7c-4b82-aa26-fdc466ee3308">


Please advise.

Reply:
Does using `!important` help.  Though we avoid using `!important` , mixins should probably treated as exceptions in that sense.